web/src/components/RunDetailAutomateMethods/methods/CLICommand/SwitchControl.tsx

```diff
@@ -0,0 +1,19 @@
+import {Switch} from 'antd';
+import * as S from './CliCommand.styled';
+
+interface IProps {
+  text: string;
+  value: boolean;
+  onChange(value: boolean): void;
+}
+
+const SwitchControl = ({value, onChange, text}: IProps) => {
+  return (
+    <S.SwitchContainer>
+      <Switch onChange={onChange} checked={value} />
+      {text}
```

Comment:
Minor: here you can add an `id` to the switch component and a `<label htmlFor="id">{text}</label>`. With this you can use the switch by clicking on the text.